config/manifests/bases/saas-operator.clusterserviceversion.yaml

```diff
@@ -2759,6 +2762,23 @@ spec:
       - description: Console specific configuration options
         displayName: Console
         path: console
+      - description: Image specification for the Sphinx component. Defaults to system
```

Comment:
```suggestion
      - description: Image specification for the Console component. Defaults to system
```

Reply:
Fixed in 91e729a0f4cd3e1b4006170bd2fa302dbf1c3768